src/Utils/Validate.js

```diff
@@ -1,14 +0,0 @@
-import {
-  validateRequired,
-  validateURL,
-} from '@folio/stripes-acq-components';
-
-const isURLValid = (value) => {
-  const validateRequiredMessage = validateRequired(value);
-
-  if (validateRequiredMessage) return validateRequiredMessage;
-
-  return validateURL(value);
-};
-
-export { isURLValid };
```

Comment:
Is `"valid.isURLValid": "Invalid URL"` from translations connected to this?

Reply:
yeah, it is. Do you suggest to remove those?

Reply:
I don't know how it works yet. Probably

Reply:
we use https://formatjs.io/docs/react-intl/components#formattedmessage, so localization strings keeps under `translations` dir. Since I'm switching to functions from stripes-acq-components, actually we don't need anymore locale strings in ui-organizations.
We just have no practice to clear them as a rule, but no harm of deleting obsolete keys there I suppose